L1Trigger/L1TTrackMatch/python/L1TkMETAnalyser_cfi.py

```diff
@@ -1,10 +1,10 @@
 import FWCore.ParameterSet.Config as cms
-from L1Trigger.L1TTrackMatch.L1TrackerEtMissEmulatorProducer_cfi import L1TrackerEmuEtMiss
-from L1Trigger.L1TTrackMatch.L1TrackerEtMissProducer_cfi import L1TrackerEtMiss
+from L1Trigger.L1TTrackMatch.L1TrackerEtMissEmulatorProducer_cfi import l1tTrackerEmuEtMiss
+from L1Trigger.L1TTrackMatch.L1TrackerEtMissProducer_cfi import l1tTrackerEtMiss
 
 L1TkMETAnalyser = cms.EDAnalyzer('L1TkMETAnalyser',
-  TrackMETInputTag = cms.InputTag("L1TrackerEtMiss",L1TrackerEtMiss.L1MetCollectionName.value()),
-  TrackMETEmuInputTag = cms.InputTag("L1TrackerEmuEtMiss",L1TrackerEmuEtMiss.L1MetCollectionName.value()),
+  TrackMETInputTag = cms.InputTag("l1tTrackerEtMiss",l1tTrackerEtMiss.L1MetCollectionName.value()),
+  TrackMETEmuInputTag = cms.InputTag("l1tTrackerEmuEtMiss",l1tTrackerEmuEtMiss.L1MetCollectionName.value()),
```

Comment:
Another case of dependency between `cfi` files.

Reply:
Fixed